roles/maintenance_10_linux/tasks/main.yml

```diff
@@ -113,6 +113,40 @@
     age: "-{{ 60 * 24 * 7 * 2 }}"  # two weeks
     exclude: "{{ linux_allowed_large_files + linux_additional_allowed_large_files }}"
 
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Disk: logrotate: Are there files in /var/log that are not rotated?"
+  ansible.builtin.shell: "cat /var/lib/logrotate/logrotate.status"
```

Comment:
Any reason this can't be done with `ansible.builtin.slurp`?

If you need to keep the `cat`, please use `command` rather than `shell`, and add a `changed_when: false` line to the task.